net/cookies/canonical_cookie.cc

```diff
@@ -162,15 +162,11 @@ Time CanonicalCookie::CanonExpiration(const ParsedCookie& pc,
   // First, try the Max-Age attribute.
   uint64_t max_age = 0;
   if (pc.HasMaxAge() &&
-#ifdef STARBOARD
-      SbStringScanF(
-#else
 #ifdef COMPILER_MSVC
```

Comment:
Is it possible for COMPILER_MSVC to be true on win32 builds (or do we never set it)? I imagine we'd want windows to share the same API as posix for now?

Reply:
If we're leaning in the direction of adopting win32 upstream code, this is fine, otherwise we should change it. Ideally we'll get to a point where windows also runs some sort of api leak detector and we should decide based on whether or not we expect this to be part of the api. I'll leave it up to you and Yijia for which way you're leaning

Reply:
We haven't decided what we want to do for win32, and don't want it to be blocking for now. I documented this in the bug.